lib/rules/web/admin_console/4.2/monitoring_alerts.xyaml

```diff
@@ -0,0 +1,112 @@
+click_icon_index_button:
+  element:
+    selector:
+      xpath: (//button[@aria-label='<aria_label_text>'])[<index>]
+    op: click
+set_placeholder_input_value:
+  element:
+    selector:
+      xpath: (//input[@placeholder='<placeholder_text>'])[<index>]
+    op: send_keys <input_value>
+    type: input
+clear_label_input_value:
+  elements:
+  - selector:
+      xpath: //label[text()='<label_text>']/..//input
+    op: clear
+    type: input
+set_label_input_value:
+  element:
+    selector:
+      xpath: //label[text()='<label_text>']/..//input
+    op: send_keys <input_value>
+    type: input
+check_div_text:
+  element:
+    selector:
+      xpath: //div[contains(text(),'<text>')]
+
+open_alert_detail:
+  action: click_alert_link_with_text
+silence_alert_from_action:
+  action: click_actions_button
+  action: click_silence_alert_button
+  action: click_create_button
+
+click_alert_link_with_text:
+  element:
+    selector:
+      xpath: //a[text()='<alert_name>']
+    op: click
+    timeout: 60
+click_actions_button:
+  element:
+    selector:
+      xpath: //button[@aria-label='Actions'][@data-test-id='actions-menu-button']
+    op: click
+click_silence_alert_button:
+  params:
+    button_text: Silence Alert
+  action: click_button_text
+
+expire_alert_from_detail: 
+  action: click_first_action_icon
+  action: click_expire_alert_button
+click_first_action_icon:
+  params:
+    aria_label_text: Actions 
+    index: 1
+  action: click_icon_index_button
+click_expire_alert_button:
```

Comment:
I see ` click_expire_alert_button`  and ` click_expire_silence_button` are using the same definition, what's the purpose of it?

Reply:
deleted click_expire_silence_button